### ISSUE TITLE
Unknown column 'typeId' in 'where clause'

### ISSUE THREAD

**Issue by dustinwalker** [nystudio107/craft-seomatic#641]
Upgrading from Craft 3.3.20.1 and SEOMatic 3.2.47 directly to Craft 3.4.20 and SEOMatic 3.3.3 causes the upgrade to fail. Happy to email composer.json & .lock + database if helpful.

If I first lock SEOMatic to 3.2.x, upgrade Craft to 3.4.x then upgrade SEOMatic it seems successful so not a major issue on our end but thought you should know. Direct upgrade I get this error: 

```
Database Exception: SQLSTATE[42S22]: Column not found: 1054 Unknown column 'typeId' in 'where clause'
The SQL being executed was: SELECT *
FROM craft_seomatic_metabundles
WHERE (sourceBundleType='section') AND (sourceId=1) AND (sourceSiteId=1) AND (typeId IS NULL)
LIMIT 1
Migration: craft\migrations\m190913_152146_update_preview_targets
Output:
Exception: SQLSTATE[42S22]: Column not found: 1054 Unknown column 'typeId' in 'where clause'
The SQL being executed was: SELECT *
FROM craft_seomatic_metabundles
WHERE (sourceBundleType='section') AND (sourceId=1) AND (sourceSiteId=1) AND (typeId IS NULL)
LIMIT 1
```

**Comment by khalwat** [comment]
Are you using an SEO Settings field @dustinwalker ? If so, this is fixed in the soon to be released 3.3.4.

You can try it now by setting your semver to `dev-develop`

**Comment by dustinwalker** [comment]
Ah, yes. I think we are (we inherited this particular site.) I'll give dev-develop a shot, thanks for the quick follow-up and apologies if I missed the probably in another issue.
